frontend/packages/knative-plugin/integration-tests/features/Knative/actions-on-knative-revision.feature

```diff
@@ -149,9 +149,27 @@ Feature: Perform actions on knative revision
 
    @smoke
    Scenario: Delete Revision not possible for the service which contains one revision : Kn-03-TC12
-      Given knative service named "<service_name>" is higlighted on topology page
+      Given knative service named "nodejs-ex-git-2" is higlighted on topology page
       When user right clicks on the revision of knative service "nodejs-ex-git-2" to open the context menu
       And user selects "Delete Revision" option from knative revision context menu
       Then modal with "Unable to delete revision" appears
       And modal contains message as "You cannot delete the last Revision for the Service."
       And modal should get closed on clicking OK button
+
+
+   @regression
+   Scenario: Revison created with successful build
+      Given user is on Import from git page
+      When user creates knative service with name "nodejs-ex-git-1"
+      And user waits for the build to complete
+      And user reruns the build
+      Then user can see two revision created
```

Comment:
@sanketpathak does this happen now ? https://issues.redhat.com/browse/BUILD-100

same applies to pipeline re run

Reply:
The build issue persist but this gherkin script is written according to the Epic mentioned and it will be taken account when this epic will complete